HW10-hibernate/src/main/java/ru/otus/core/model/User.java

```diff
@@ -0,0 +1,98 @@
+package ru.otus.core.model;
+
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
```

Comment:
```suggestion
    @Column(name = "user_id", nullable = false)
```

Лучше единообразно как `add_id`, `phone_id`